### PR TITLE
chore: it restores inquirer for prompting external commands

### LINKED ISSUE #1200: The command moved into a separate package: undefined when run webpack serve
**What is the current behavior?**

image
I used `webpack serve`，but it didn't work.
This is my package dependencies:

```json
  "devDependencies": {
    "webpack": "^5.0.0-beta.12",
    "webpack-cli": "^4.0.0-beta.1",
    "webpack-dev-server": "^3.10.2"
  }
```

**To Reproduce**
1. run `npx webpack serve`

**Expected behavior**
Run webpack-dev-server or give correct tips about what should be installed

### PR DESCRIPTION
**Pull request by ematipico**
**What kind of change does this PR introduce?**

Fixes #1200 

**Did you add tests for your changes?**
Not needed

**If relevant, did you update the documentation?**
Not for now

**Summary**
A fix for the bug was already there but we have uncomment the code and make a new release

**Does this PR introduce a breaking change?**

No

**Other information**

### CODE REVIEW

File: lib/commands/external.js [hunk 1]
```diff
@@ -20,47 +22,46 @@ class ExternalCommand {
...
+        const { installConfirm } = await prompt([
+            {
+                type: 'confirm',
+                name: 'installConfirm',
+                message: question,
+                default: 'Y',
+                choices: ['Yes', 'No', 'Y', 'N', 'y', 'n'],
+            },
+        ]);
+        if (installConfirm === true) {
```
Comment (jamesgeorge007) [inline lib/commands/external.js]:
```suggestion
        if (installConfirm) {
```

Comment (jamesgeorge007) [reply lib/commands/external.js]: This one seems to be a redundant check :thinking:

Comment (ematipico) [reply lib/commands/external.js]: My question: is `installConfirm` always a boolean?

Comment (jamesgeorge007) [reply lib/commands/external.js]: Yes it is.

**Comment by ematipico** [comment]
Maybe later. For now we skipped it.

**Comment by rishabh3112** [comment]
Lets merge it when CI passes

File: lib/commands/external.js [hunk 2]
```diff
@@ -20,47 +22,46 @@ class ExternalCommand {
...
+        }
+        // eslint-disable-next-line require-atomic-updates
+        process.exitCode = -1;
+    }
 
     static async run(name, ...args) {
-        let pkgLoc = ExternalCommand.validateEnv(name);
-        const scopeName = '@webpack-cli/' + name;
+        let pkgLoc = ExternalCommand.checkIfPackageExists(name);
+        const scopeName = packagePrefix + '/' + name;
```
Comment (jamesgeorge007) [inline lib/commands/external.js]: Wanna use template literals here?

**Pull request merged**